<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>enter_form</class>
 <widget class="QMainWindow" name="enter_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red; font-size: 14px;</string>
    </property>
    <property name="text">
     <string>Логин</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red; font-size: 14px;</string>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="login_line">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red; font-size: 14px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_line">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red; font-size: 14px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enter_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: red;
	font-size: 16px;
}

QPushButton:hover {
    color: white;
}</string>
    </property>
    <property name="text">
     <string>Войти</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
